entropy/README.md

```diff
@@ -15,8 +15,50 @@ The entropy server
   bucket for use by verifier(s)
 - Serves it up over http
 
+The [figure below](#fig-entropy-server) illustates the server in a deployment setup.
+
 Note that TLS termination and CDN use is required to expose the server publicly.
 
+<div align = "center" markdown = 1 id="fig-entropy-server">
```

Comment:
Does this need a return arrow from the entropy server to the requesting hotspot as well? Implies that the data flow is one-way at present

Reply:
fixing